overlord/servicestate/servicestate.go

```diff
@@ -332,5 +333,19 @@ func SnapServiceOptions(st *state.State, instanceName string) (opts *wrappers.Sn
 			break
 		}
 	}
+
+	// also check for quota group for this instance name
+	allGrps, err := AllQuotas(st)
```

Comment:
for the usage in configcore this feels quite wasteful, maybe until we can move that code over here, we can have this take an optional quota group map?

Reply:
done